src/js/themes/hpe.js

```diff
@@ -2270,6 +2294,7 @@ const buildTheme = (tokens, flags) => {
         },
       },
       info: {
+        icon: Info,
```

Comment:
I believe this should be `StatusInfo`. See https://github.com/grommet/grommet-theme-hpe/wiki/Migrating-from-HPE-Theme-7.x-to-8.x#status-backgrounds

```suggestion
        icon: StatusInfo,
```